src/styles/theme/oc-sidebar.scss

```diff
@@ -0,0 +1,122 @@
+.oc-sidebar {
+  width: $width-medium;
+  height: 100vh;
+  max-height: 100vh;
+  padding: $gutter 0;
+  background-color: $primary-background;
+  overflow: hidden auto;
+  box-sizing: border-box;
+
+  &-content-wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 100vw;
```

Comment:
looks weird to have width bigger than max-width. It works - but you could just set it to the same value as max-width (`$width-medium`).

Reply:
I did this in order to not have to re-assign the value to 100vw in the fixed later on. This way I assign it here once and only care about the max-width. But I am fine also with changing it 😉